Update case studies (#169)
ambanum/OpenTermsArchive.org: src/translations/en/case-studies.mdx

### Protonmail clarifies its privacy policies

On September 6, 2021, ProtonMail <a target="_blank" rel="noopener" href="https://github.com/OpenTermsArchive/contrib-versions/commit/510c7d65e4254e1d53221b139d7e25bb2b990510">amended</a> its privacy policies so as to clarify the effects of Swiss laws (where the company is based) on the handling of user data. So, while the company claims to offer a particularly secure email service, it may find itself "forced to share the IP address" of its users with law enforcement in criminal investigations.

> The day before the change, the website <a target="_blank" rel="noopener" href="https://paris-luttes.info/recit-policier-de-sainte-marthe-15258?lang=fr">Paris-Luttes.info</a> ands various <a target="_blank" rel="noopener" href="https://mobile.twitter.com/OnEstLaTech/status/1434576598418796549">Internet users</a> revealed that ProtonMail had communicated the IP addresses of users linked to the Youth for Climate group to the Swiss justice system, which was itself requested by the French police <a target="_blank" rel="noopener" href="https://www.numerama.com/tech/736940-protonmail-transmet-des-adresses-ip-a-la-police-4-questions-pour-comprendre-la-polemique.html">via Europol</a>. This change mirrors what the company had indicated on <a target="_blank" rel="noopener" href="https://www.reddit.com/r/ProtonMail/comments/pil6xi/climate_activist_arrested_after_protonmail/hbqha63">Reddit</a>.

Reviewer: ```suggestion
The day before the change, the website <a target="_blank" rel="noopener" href="https://paris-luttes.info/recit-policier-de-sainte-marthe-15258?lang=fr">Paris-Luttes.info</a> and various <a target="_blank" rel="noopener" href="https://mobile.twitter.com/OnEstLaTech/status/1434576598418796549">Internet users</a> revealed that ProtonMail had communicated the IP addresses of users linked to the Youth for Climate group to the Swiss justice system. These addresses had been requested by the French police <a target="_blank" rel="noopener" href="https://www.numerama.com/tech/736940-protonmail-transmet-des-adresses-ip-a-la-police-4-questions-pour-comprendre-la-polemique.html">via Europol</a>. This change mirrors what the company had indicated on <a target="_blank" rel="noopener" href="https://www.reddit.com/r/ProtonMail/comments/pil6xi/climate_activist_arrested_after_protonmail/hbqha63">Reddit</a>.
```

Author: done

---

> ### Alibaba adds disease outbreaks to its disclaimer.

Reviewer: Use consistent title punctuation 🙂 

```suggestion
### Alibaba adds disease outbreaks to its disclaimer
```

> Covid's epidemic can be read right into Alibaba's terms of service: since May 6, 2020, the marketplace <a target="_blank" rel="noopener" href="https://github.com/ambanum/CGUs-data/commit/37503cb23">specifies</a> in cases of force majeure in which it is not responsible for order delays or failures.

Reviewer: ```suggestion
The impact of the Covid-19 pandemic can be read right into Alibaba's terms of service: since May 6, 2020, the marketplace <a target="_blank" rel="noopener" href="https://github.com/ambanum/CGUs-data/commit/37503cb23">specified</a> epidemics as cases of force majeure in which it is not responsible for order delays or failures.
```

Author: Done

---

> ### Pinterest removes Privacy Shield mentions without updating the modification date

Reviewer: ```suggestion
### Pinterest removes Privacy Shield notice without updating its modification date
```

Author: Done

> Pinterest has <a target="_blank" rel="noopener" href="https://github.com/ambanum/CGUs-data/commit/a53e891ba">removed</a> a lengthy explanation of its data processing arrangements, which it said it was carrying out under the Privacy Shield then in place between the European Union and the United States. Until then, the company was committed to resolving any complaints about personal data processing and verifying as much as possible that its partners were also in compliance with the Privacy Shield. On July 21, 2020, this passage was replaced by a simple sentence stating that personal data from the European Economic Area could be processed outside, without a commitment to manage complaints. The platform did not indicate in its "summary of recent changes" at the top of the document that it had made any such changes.

Reviewer: ```suggestion
Pinterest has <a target="_blank" rel="noopener" href="https://github.com/ambanum/CGUs-data/commit/a53e891ba">removed</a> a lengthy explanation of its data processing arrangements, which it said it was carrying out under the Privacy Shield then in place between the European Union and the United States. Until then, the company was committed to resolving any complaints about personal data processing and verifying as much as possible that its partners were also in compliance with the Privacy Shield. On July 21, 2020, this part was replaced by a simple sentence stating that personal data from the European Economic Area could be processed outside of it, without a commitment to manage complaints. The platform did not indicate in its "summary of recent changes" at the top of the document that it had made any such changes.
```

Author: Done

> These can be read in light of the court ruling five days earlier by the Court of Justice of the European Union, which <a target="_blank" rel="noopener" href="https://www.cnil.fr/fr/invalidation-du-privacy-shield-les-suites-de-larret-de-la-cjue">invalidated</a> the extra-continental data transfer regime previously in place.

Reviewer: ```suggestion
These can be read in light of the court ruling five days earlier by the Court of Justice of the European Union, which <a target="_blank" rel="noopener" href="https://www.cnil.fr/fr/invalidation-du-privacy-shield-les-suites-de-larret-de-la-cjue">invalidated</a> the bulk data transfer regime previously in place.
```

---

### Qwant details data shared with Microsoft

> On November 2, 2021, Qwant <a target="_blank" rel="noopener" href="https://github.com/OpenTermsArchive/contrib-versions/commit/549e959ef7671a194b9bedba8d12c8031c39b922">specifies</a> in its privacy policy, the type of data it passes on to its partner Microsoft "for the security and reliability of its services." Among the data that the service can share with Bing, the search engine of the American giant, are the keywords searched, the type and version of the browser used, or an identifier calculated from the IP address and browser information.

Reviewer: ```suggestion
On November 2, 2021, Qwant <a target="_blank" rel="noopener" href="https://github.com/OpenTermsArchive/contrib-versions/commit/549e959ef7671a194b9bedba8d12c8031c39b922">specified</a> in its privacy policy the type of data it passes on to its partner Microsoft "for the security and reliability of its services." Among the data that the service can share with Bing, the search engine of the American giant, are the keywords searched, the type and version of the browser used, or an identifier derived from the IP address and browser information.
```

Author: Done

---

### Instagram blocks data retrieval and clarifies what information it retrieves

In November 2021, Instagram made several updates to its <a target="_blank" rel="noopener" href="https://github.com/OpenTermsArchive/contrib-versions/commit/9998cb21b69a222540491a81dd05be19a9785891">community rules</a>, its <a target="_blank" rel="noopener" href="https://github.com/OpenTermsArchive/contrib-versions/commit/9aaf5dbfc3b34d45d640b0c7f96dce3598c7bcd2">terms of use</a> and its <a target="_blank" rel="noopener" href="https://github.com/OpenTermsArchive/contrib-versions/commit/927ceba3865d79cd1b123d9f4b9928b8e997070c">privacy policy</a>. Among the notable developments, the platform explicitly prohibits the sale or exchange of "non-medical or pharmaceutical substances." It says it will remove all posts promoting such products, either for sale or to indicate personal consumption, except in a healing context.

> In its updated terms of service, Instagram prohibits creating accounts or using them in such a way as to retrieve information without permission, especially in an automated way. This could have an effect on the work of researchers like those of the NGO Algorithm Watch: a research project of the latter had already been closed by Instagram <a target="_blank" rel="noopener" href="https://algorithmwatch.org/en/instagram-research-shut-down-by-facebook/">by summer 2021</a>. The platform also specifies the prohibition of disseminating another user's personal information, including data or publications that are the intellectual property of another Internet user.

Reviewer: ```suggestion
In its updated terms of service, Instagram prohibits creating accounts or using them in such a way as to retrieve information without permission, especially in an automated way. This could have an effect on the work of researchers like those of the NGO Algorithm Watch, a research project of which having already been <a target="_blank" rel="noopener" href="https://algorithmwatch.org/en/instagram-research-shut-down-by-facebook/">closed by Instagram</a> in the summer of 2021. The platform also prohibits disseminating other users’ personal information.
```

Author: done

> Finally, the social network elaborates on the type of data it collects in its privacy policies: metadata about the tools used by the user, what the user sees or records with his or her camera, the publications and accounts with which he or she interacts, or the frequency, duration and time of day he or she performs these activities. Meta's partners receive some or all of the data when their services are used.

Reviewer: ```suggestion
Finally, the social network elaborates on the type of data it collects in its privacy policies: metadata about the tools used by the user, what users see or record with their camera, the posts and accounts with which they interact, as well as the frequency, duration and time of day they perform such activities. Meta's partners receive some or all of that data when their services are used.
```

Author: done
